app/containers/App/index.tsx

```diff
@@ -36,37 +36,40 @@ const CustomLayout = styled(Layout)`
 
 export function App() {
   return (
-    <ThemeProvider theme={theme}>
+    <CustomLayout>
       <Header />
-      <CustomLayout>
-        <Sidebar />
-        <Layout.Content>
-          <ItunesApiComponent />
-          <For
-            ParentComponent={(props) => <Switch {...props} />}
-            of={map(Object.keys(routeConfig))}
-            renderItem={(routeKey, index) => {
-              const Component = routeConfig[routeKey].component;
-              return (
-                <Route
-                  exact={routeConfig[routeKey].exact}
-                  key={index}
-                  path={routeConfig[routeKey].route}
-                  render={(props) => {
-                    const updatedProps = {
-                      ...props,
-                      ...routeConfig[routeKey].props
-                    };
-                    return <Component {...updatedProps} />;
-                  }}
-                />
-              );
-            }}
-          />
-          <GlobalStyle />
-        </Layout.Content>
-      </CustomLayout>
-    </ThemeProvider>
+      <ItunesApiComponent />
+    </CustomLayout>
+    // <ThemeProvider theme={theme}>
+    //   <Header />
+    //   <CustomLayout>
+    //     <Sidebar />
+    //     <Layout.Content>
+    //       <For
+    //         ParentComponent={(props) => <Switch {...props} />}
+    //         of={map(Object.keys(routeConfig))}
+    //         renderItem={(routeKey, index) => {
+    //           const Component = routeConfig[routeKey].component;
+    //           return (
+    //             <Route
+    //               exact={routeConfig[routeKey].exact}
+    //               key={index}
+    //               path={routeConfig[routeKey].route}
+    //               render={(props) => {
+    //                 const updatedProps = {
+    //                   ...props,
+    //                   ...routeConfig[routeKey].props
+    //                 };
+    //                 return <Component {...updatedProps} />;
+    //               }}
+    //             />
+    //           );
+    //         }}
+    //       />
+    //       <GlobalStyle />
+    //     </Layout.Content>
+    //   </CustomLayout>
+    // </ThemeProvider>
```

Comment:
@prabhat-wednesday please remove whatever is not required & also look into this files that what are different things that are commented but useful.